### PR TITLE
SS-91: extend the info-level log + add fix for favicon error in log

### PR DESCRIPTION
**Pull request by eivinde**
extend the info-level log + add fix for favicon error in log+ fix typo

### CODE REVIEW

File: service/incremental-jsonsystem.py [hunk 1]
```diff
@@ -73,6 +73,11 @@ def make_session(self):
             session.headers = self._config['headers']
         return session
 
+# to remove favicon not found errormessages in the log
+@app.route('/favicon.ico')
```
Comment (fluan) [inline service/incremental-jsonsystem.py:77]: Do we need it?

File: service/incremental-jsonsystem.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from flask import Flask, request, abort
+from flask import Flask, request, abort,send_from_directory
```
Comment (fluan) [inline service/incremental-jsonsystem.py]: need a space after the comma in "abort,send_from_directory"

**Review by fluan — APPROVED** [review]

File: service/incremental-jsonsystem.py (hunk 1 above)
Comment (eivinde) [reply service/incremental-jsonsystem.py:77]: we do not "need" it, but it removes annoying error messages from the logs

File: service/incremental-jsonsystem.py (hunk 2 above)
Comment (eivinde) [reply service/incremental-jsonsystem.py]: fixed

**Pull request merged**